server/views/makeAReferral/needsAndRequirements.njk

```diff
@@ -35,7 +35,9 @@
         {{ govukTextarea(whenUnavailableTextareaArgs) }}
         {% endset -%}
         {{ govukRadios(responsibilitiesRadiosArgs(whenUnavailableHtml)) }}
-
+        {% if reasonForChangeTextAreaArgs %}
+            {{ govukTextarea(reasonForChangeTextAreaArgs) }}
+        {% endif %}
         {{ govukButton({ text: "Save and continue", preventDoubleClick: true }) }}
```

Comment:
Link text doesn't match the design (+ missing back link, and check with Helen - other amends have a Cancel and go back link)